feat: Added `toggle-mode`, allowing toggle-on and toggle-off (#2555)
zmkfirmware/zmk: docs/docs/keymaps/behaviors/key-toggle.md

You can use any keycode that works for `&kp` as parameter to `&kt`, however, [modified keys](../modifiers.mdx#modifier-functions) such as `LA(A)` will be toggled based on the status of the base keycode (in this case `A`).
In other words, modifiers are ignored when determining whether or not the key is currently pressed.

### Configuration

#### Toggle mode

If you wish to ensure that a key is pressed or released, rather than merely toggling, then you can do so with the `toggle-mode` property.
Define a new behavior and assign `"on"` or `"off"` to `toggle-mode`:

```dts
/ {
    behaviors {
        kt_on: key_toggle_on_only {
            compatible = "zmk,behavior-key-toggle";
            #binding-cells = <1>;
            display-name = "Key Toggle On";
            toggle-mode = "on";
        };
    };
};
```

> You can then use `&kt_on` in place of `&kt` whenever you wish to only toggle a key on, and not toggle it off. The `"off"` version is analogous.

Reviewer: ```suggestion
You can then use `&kt_on` in place of `&kt` whenever you wish to only toggle a key on, and not toggle it off. An `"off"` version of the behavior can be defined similarly.
```
